db/models/product.js

```diff
@@ -7,14 +7,50 @@ const bcrypt = require('bcryptjs')
 module.exports = db => db.define('products', {
   name: {
     type: Sequelize.STRING,
```

Comment:
This is going to fail because we never defined Sequelize!  On line 5 switch {STRING,VIRTUAL} object to Sequelize = require('sequelize')!